speckle_connector/converter/to_native.rb

```diff
@@ -119,9 +119,118 @@ def _speckle_mesh_to_native_mesh(mesh, entities)
       native_mesh.add_polygon(indices.map { |index| points[index] })
     end
     entities.add_faces_from_mesh(native_mesh, 4, material_to_native(mesh["renderMaterial"]))
+    remove_coplanar_faces(entities)
     native_mesh
   end
 
+  # Removes coplanar entities from active_model.
+  # @param entities [Sketchup::Entities] entities to remove edges between that make entities coplanar.
+  def remove_coplanar_faces(entities)
```

Comment:
i've tested this out with test streams from various applications and all seems to work as expected - thanks so much for the contribution! it looks great 🚀

it all looks pretty logical and i appreciate the commented code. a couple of tiny nitpicks:

1. i would rename or more explicitly document `detect_edges_have_coplanar_faces` since it doesn't just _detect_ but actually _erases_ edges which is not clear from the name or the method docstring which could be confusing if someone wanted to use the method independently
2. i would amend the docstring for `remove_coplanar_faces` to "Removes coplanar entities from _the given entities_" for clarity since the entities don't have to be the entire `active_model` entitites. also, maybe we rename this method to `merge_coplanar_faces` to be a bit more descriptive?

Reply:
Thanks for your feedbacks! Your suggestions are resolved! See: https://github.com/specklesystems/speckle-sketchup/pull/58/commits/cd3e6c68b2e9fcf86064144cc16195f784727f95 https://github.com/specklesystems/speckle-sketchup/pull/58/commits/5b9d1729286ca54affe3a3d3267c51b63c861465

Reply:
looks great - thanks a lot!